runHelpers.ts

```diff
@@ -10,7 +10,8 @@ import { parseOptions } from "https://deno.land/x/amqp@v0.23.1/src/amqp_connect_
 
 import { Chain } from "npm:viem";
 
-import { type Prisma, PrismaClient } from "./prisma-shim.ts";
+import type Prisma from "./prisma-shim.ts";
+import { PrismaClient } from "./prisma-shim.ts";
```

Comment:
(This is just my preference and there's actually no need to change here, you can just ignore this comment) I like type-only imports, but if there's no way to do type-only import and named import in single line, I would choose single import between them.

Reply:
Actually this is because when there's a type keyword you cannot do both default import and named import. I personally think if it's possible to do a type import (which is a constraint, in a way) it should be better to do a type import, even if it makes us to separate the statement, as it narrows down what we can and cannot do (which might help avoid mistakes.)